pypeit/scripts/run_to_calibstep.py

```diff
@@ -0,0 +1,100 @@
+"""
+Script to run to a single calibration step for an input frame
+
+.. include common links, assuming primary doc root is up one directory
+.. include:: ../include/links.rst
+"""
+
+from pypeit.scripts import scriptbase
+
+class RunToCalibStep(scriptbase.ScriptBase):
+
+    # TODO: Combining classmethod and property works in python 3.9 and later
+    # only: https://docs.python.org/3.9/library/functions.html#classmethod
+    # Order matters.  In python 3.9, it would be:
+    #
+    # @classmethod
+    # @property
+    #
+    # Because we're not requiring python 3.9 yet, we have to leave this as a
+    # classmethod only:
+    @classmethod
+    def name(cls):
+        """
+        Return the name of the executable.
+        """
+        return 'pypeit_run_to_calibstep'
+
+    @classmethod
+    def get_parser(cls, width=None):
+        import argparse
+
+        parser = super().get_parser(description='Run PypeIt to a single calibration step for an input frame',
+                                    width=width, formatter=argparse.RawDescriptionHelpFormatter)
```

Comment:
I think you're better off with the default formatter; the reason this is used in `run_pypeit.py` is so that the parser can handle the usage string.